macros/schema_tests/table_shape/expect_table_row_count_to_equal.sql

```diff
@@ -1,4 +1,4 @@
-{%- macro test_expect_table_row_count_to_equal(model,value,row_condition=None) -%}
+{%- macro test_expect_table_row_count_to_equal(model, value, row_condition) -%}
     {{ adapter.dispatch('test_expect_table_row_count_to_equal', packages=dbt_expectations._get_namespaces()) (model,value,row_condition=None) }}
```

Comment:
Hey! Sorry, small thing: could we add spaces here as well like you did above?
Also, please keep the `None` default in the macro parameters, but take it out of the dispatch call.
```
{%- macro test_expect_table_row_count_to_equal(model, value, row_condition=None) -%}    
    {{ adapter.dispatch('test_expect_table_row_count_to_equal', packages=dbt_expectations._get_namespaces()) (model, value, row_condition) }}
```